test/tsconfig.json

```diff
@@ -13,11 +13,6 @@
     "esModuleInterop": true,
     "declaration": false,
     "noEmit": true,
-    "baseUrl": ".",
-    "paths": {
-      "msw": ["../"],
-      "msw/*": ["../*"]
-    },
```

Comment:
This prevents `package.json#exports` lookup, see: https://github.com/microsoft/playwright/blob/a84ba4db691e7d2a51ff5b2460c9946d406edfc0/packages/playwright/src/util.ts#L365-L375

Reply:
You might have to *build* before you run your tests now. If this is a problem let's figure out the solution to this once we make everything functional

Reply:
Oh, and I'd really just encourage you to get rid of the other `tsconfig.json#paths` you have. My rule of thumb when it comes to `tsconfig.json#paths` is simple: "just don't". I don't think it's worthwhile to save yourself from a few relative paths - especially in the context of a *library*

Reply:
> You might have to build before you run your tests now.

That was always the case for browser tests. I believe it wasn't the case for node tests but that's a small price to pay for having aliases working.